WordPress/src/main/java/org/wordpress/android/ui/posts/PostListAction.kt

```diff
@@ -48,6 +50,9 @@ fun handlePostListAction(
         is PostListAction.NewPost -> {
             ActivityLauncher.addNewPostForResult(activity, action.site, action.isPromo, POST_FROM_POSTS_LIST)
         }
+        is PostListAction.NewStoryPost -> {
+            ActivityLauncher.addNewStoryForResult(activity, action.site, STORY_FROM_POSTS_LIST)
```

Comment:
I think we'll need to show the media picker here instead, something like this:

```
            ActivityLauncher.showPhotoPickerForResult(
                    activity,
                    WP_STORIES_MEDIA_PICKER,
                    action.site,
                    null // this is not required, only used for featured image in normal Posts
            )
```
similar to what method `handleNewStoryAction()` does in WPMainActivity. Also, we'll need to process `onActivityResult()` to get the user's selection and such.

Reply:
Hi @mzorz I resolved this in the follow-up comments below. Let me know your thoughts.